src/databricks/labs/ucx/queries/assessment/main/01_3_all_tables.sql

```diff
@@ -13,5 +13,9 @@ SELECT CONCAT(`database`, '.', name) AS name,
            ELSE "EXTERNAL"
        END AS storage,
        IF(format = "DELTA", "Yes", "No") AS is_delta,
-       location
-FROM $inventory.tables
+       location,
+       size_in_bytes
```

Comment:
Probably convert it to MB/GB/UNKNOWN with case-when

Reply:
Done